scripts/contractInteraction/contract_interaction.py

```diff
@@ -35,8 +35,8 @@ def main():
     
     #executeOnMultisig(1071)
    
-    #confirmWithMS(1064)
-    #checkTx(1077)
+    #confirmWithMS(1139)
```

Comment:
Why is this activated?

Reply:
just signing leftovers
not material while commented out as the PR is not gonna be merged